vmdb/app/assets/javascripts/miq_application.js

```diff
@@ -2,55 +2,81 @@
-  if (miqDomElementExists('col1')) miqInitDashboardCols();  // Initialize the dashboard column sortables
-  if (miqDomElementExists('widget_select_div')) miqInitWidgetPulldown();  // Initialize the dashboard widget pulldown
-
+  // Initialize the dashboard column sortables
+  if (miqDomElementExists('col1')) {
+    miqInitDashboardCols();
+  }
+  // Initialize the dashboard widget pulldown
+  if (miqDomElementExists('widget_select_div')) {
+    miqInitWidgetPulldown();
+  }
+  // Need this since IE will not run JS correctly until after page is loaded
   if (typeof init_dhtmlx_layout != "undefined") {
-    miqInitDhtmlxLayout();  // Need this since IE will not run JS correctly until after page is loaded
+    miqInitDhtmlxLayout();
   }
 
-  $(document).mousemove(function(e){
-    miqGetMouseXY(e.pageX, e.pageY)
+  $(document).mousemove(function(e) {
+    miqGetMouseXY(e.pageX, e.pageY);
   });
-  if (miqDomElementExists('compare_grid')) {  // Need to do this here for IE, rather then right after the grid is initialized
+
+  // Need to do this here for IE, rather then right after the grid is initialized
+  if (miqDomElementExists('compare_grid')) {
     compare_grid.enableAutoHeight(true);
     compare_grid.enableAutoWidth(true);
   }
+
   miqBuildCalendar();
   miqLoadCharts();
-  if (typeof miqLoadTL == "function"){
+
+  if (typeof miqLoadTL == "function") {
     miqLoadTL();
-    if (miq_timeline_filter){
-      performFiltering(tl, [0,1]);
+    if (miq_timeline_filter) {
+      performFiltering(tl, [0, 1]);
     }
   }
 
-  if (typeof miqInitGrids == "function") miqInitGrids();            // Initialize dhtmlxgrid control
-
-  if (typeof miqInitToolbars == "function") miqInitToolbars();      // Init the toolbars
-
-  if (typeof miqEditor != "undefined") miqEditor.refresh();         // Refresh the myCodeMirror editor
-
-  if (typeof miq_tree_focus == "string") eval(miq_tree_focus);      // Focus on certain tree node, if set
-
-  if (miqDomElementExists('clear_search'))                             // Position clear search link in right cell header
-    $('.dhtmlxInfoBarLabel:visible').append($('#clear_search')[0])  // Find the right cell header div
-
-  if (typeof miq_after_onload == "string") eval(miq_after_onload);  // Run MIQ after onload code if present
-
+  // Initialize dhtmlxgrid control
+  if (typeof miqInitGrids == "function") {
+    miqInitGrids();
+  }
+  // Init the toolbars
+  if (typeof miqInitToolbars == "function") {
+    miqInitToolbars();
+  }
+  // Refresh the myCodeMirror editor
+  if (typeof miqEditor != "undefined") {
+    miqEditor.refresh();
+  }
+  // Focus on certain tree node, if set
+  if (typeof miq_tree_focus == "string") {
+    eval(miq_tree_focus);
+  }
+  // Position clear search link in right cell header
+  if (miqDomElementExists('clear_search')) {
+    // Find the right cell header div
+    $('.dhtmlxInfoBarLabel:visible').append($('#clear_search')[0]);
```

Comment:
I don't think that the `[0]` is necessary! The selector uses a should be unique ID and so there is only one selected element to append!